ansible/roles/olm_setup/templates/osb-operators.configmap.upstream.yaml

```diff
@@ -74,9 +74,46 @@ data:
       metadata:
         name: templateservicebrokeroperator.v0.1.0
         namespace: placeholder
+        annotations:
+          alm-examples: '[{"apiVersion":"osb.openshift.io/v1alpha1","kind":"TemplateServiceBroker","metadata":{"name":"template-service-broker","namespace":"openshift-template-service-broker"},"spec":{"size":3,"version":"3.2.13"}}]'
       spec:
         displayName: Template Service Broker Operator
-        description: Template Service Broker Operator
+        description: |
+          The Template service broker implements the [Open Service Broker
```

Comment:
Think this should read "Template Service Broker" to match capitalization elsewhere.